frontend/src/components/SessionDetail/CompatSessionDetail.tsx

```diff
@@ -124,6 +92,10 @@ const CompatSessionDetail: React.FC<Props> = ({ session }) => {
       <SessionHeader to="/sessions">{data.deviceId || data.id}</SessionHeader>
       <SessionDetails
         title={t("frontend.compat_session_detail.session_details_title")}
+        deviceId={data.deviceId}
+        signedIn={parseISO(data.createdAt)}
+        lastActive={data.lastActiveAt ? parseISO(data.lastActiveAt) : undefined}
+        ipAddress={data.lastActiveIp ?? undefined}
```

Comment:
On the consent screen for compat sessions, we "fake" a scope list to have something ~coherent ; we could do the same here?

https://github.com/matrix-org/matrix-authentication-service/blob/e5a88622db81c90315d2942664c664ee795da604/templates/pages/sso.html#L34

Reply:
Yes, but I'd argue that should be a separate PR as the design doesn't explicitly state to do this only showing one comp.